src/plotGrid.jl

```diff
@@ -0,0 +1,27 @@
+import jInv.Vis.plotGrid
+
+function plotGrid(M::OcTreeMesh;kwargs...)
```

Comment:
shall we have a fall-back when the user does not have PyPlot installed? See the variable ``jInv.Vis.hasPyPlot``.